src/tools/cephfs/top/cephfs-top

```diff
@@ -943,10 +942,10 @@ class FSTop(object):
 
     def run_all_display(self):
         # clear text from the previous screen
-        if self.current_screen == FS_TOP_FS_SELECTED_APP:
+        if self.active_screen == FS_TOP_FS_SELECTED_APP:
```

Comment:
@joscollin What's the notion behind having a check before `self.header.erase()` on switching screens only in `run_all_display()` not in `self.run_display()`?

Reply:
It means the current run is a re-run from `run_display` and it needs a complete erase of the header. Please comment that code and check, you would see the previous text in the header.

Reply:
Yes, I understood. I want to ask why this check is not required in `run_display()`?

Reply:
For `run_display` the check is not required, as it's always erasing the header at the starting point. This erase is necessary for `run_display`. But an erase is not necessary for `run_all_display` when cephfs-top is launching. So basically the check saves an unnecessary erase call.

Reply:
Okay, we are saving an unnecessary call for erase when cephfs-top launches.

Reply:
yes